tests/_modules.js

```diff
@@ -0,0 +1,715 @@
+module.exports =  [{ path: '/Users/samrichard/Development/plugabilly/node_modules/ava',
```

Comment:
It's because I want a static representation to test against so I'm only testing the sorting, not testing the sorting and the meta loading